Source/ProjectFirma.Web/Views/Program/ProjectBlockListGridSpec.cs

```diff
@@ -41,7 +40,11 @@ public ProjectBlockListGridSpec(Person currentPerson, Models.Program currentProg
 
             Add(Models.FieldDefinition.ProjectIdentifier.ToGridHeaderString(),
                 x => (x.Project != null) ? x.Project.ProjectGisIdentifier : x.ProjectGisIdentifier,
-                300, DhtmlxGridColumnFilterType.Html);
+                200, DhtmlxGridColumnFilterType.Html);
+
+            Add("Notes",
+                x => (!string.IsNullOrEmpty(x.Notes)) ? x.Notes : string.Empty,
+                500, DhtmlxGridColumnFilterType.Html);

```

Comment:
FilterType.Html should probably be .Text. Html will treat the cells as HtmlString. and I think ProjectGidIdentifier is just a string

Reply:
Made the change! Good catch; I never looked at that last part of the column declaration. Notes was copied from the ProjectGisIdentifier, which was copied from the ProjectId, which could be a link rather than just straight text like the other two.